vnext/Shared/DevSupportManager.cpp

```diff
@@ -232,6 +237,37 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
+void DevSupportManager::startInspector(
+    [[maybe_unused]] const std::string &packagerHost,
+    [[maybe_unused]] const uint16_t packagerPort) {
+#ifdef HERMES_ENABLE_DEBUGGER
+  std::string packageName("RNW");
+  if (winrt::Windows::ApplicationModel::Package::Current()) {
+    packageName = winrt::to_string(winrt::Windows::ApplicationModel::Package::Current().DisplayName());
```

Comment:
>winrt::Windows::ApplicationModel::Package::Current() [](http://example.com/codeflow?start=35&length=52)

We can avoid calling the same function twice by putting its result above in the "if" statement: `if (auto currentPackage = ...) ...`